action/recipes/config.go

```diff
@@ -69,6 +69,17 @@ type CommonOpts struct {
 	//     ├── coreboot.rom
 	//     └── defconfig
 	OutputDir string `json:"output_dir" validate:"required,dirpath"`
+
+	// Specifies the path to directory where to place input files and directories inside container.
+	//   Directories listed in ContainerInputDirs and files listed in ContainerInputFiles
+	//   will be copied there.
+	ContainerInputDir string `json:"container_input_dir" validate:"dirpath"`
```

Comment:
The naming of `InputDir` vs `InputDirs` feels a bit confusing to me, given that they don't mean the same thing and one isn't just the plural form. Maybe name the first one `ContainerInputTargetDir`? Otherwise I'm open to other suggestions but the current naming feels to confusing.

Reply:
You have a valid point, I will change this

Reply:
I have changed it so that it is more coherent to the existing `ContainerOutputDirs`, `ContainerOutputFiles` and `OutputDir` settings.

And just to make sure I have added more documentation.